packages/dev/core/src/Shaders/velocity.fragement.ts

```diff
@@ -0,0 +1,13 @@
+// Do not edit.
+import { ShaderStore } from "../Engines/shaderStore";
+
+const name = "velocityVertexShader";
+const shader = `precision highp float;
#define CUSTOM_FRAGMENT_BEGIN
```

Comment:
nit: can we take the "^M" out? it seems to be how github is displaying the carriage return 😅

Reply:
this file was removed, it was not supposed to be commited